.github/workflows/image-go.yml

```
@@ -38,6 +38,12 @@ on:
         description: Version of golang used during build if specified as argument in Dockerfile
         required: false
         type: string
+      runs-as:
+        description: >
+          Defines how the built image is expected to be run, e.g service or cronjob.
+          The default is service.
+        required: false
+        type: string
```

You can use the `choice` type and provide two options (see [here](https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/))
```suggestion
      runs-as:
        description: Defines how the built image is expected to be run, e.g service or cronjob.
        type: choice
        options:
        - service
        - cronjob
```